mix.exs

```diff
@@ -43,7 +43,8 @@ defmodule ReadingTime.MixProject do
     [
       {:dialyxir, "~> 1.0", only: [:dev], runtime: false},
       {:ex_doc, "~> 0.22", only: :dev, runtime: false},
-      {:stream_data, "~> 0.5", only: :test}
+      {:stream_data, "~> 0.5", only: :test},
+      {:credo, "~> 1.5.0-rc.2", only: [:dev, :test], runtime: false}
```

Comment:
What do you think about using the stable version?
```elixir
{:credo, "~> 1.4", only: [:dev, :test], runtime: false}}
```

Also, we could move it to the top so it keeps sorted alphabetically